training.rb

```diff
@@ -0,0 +1,18 @@
+module Training
```

Comment:
These aren't all in all of the teacher/student classes.  Only group behaviors that are common to multiple classes.

Looks like you didn't actually use this -- so take it out.